pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -441,3 +508,18 @@ func (r *NetworkInfoReconciler) deleteVPCs(ctx context.Context, staleVPCs []*mod
 	deleteVPCNetworkConfigurationStatus(ctx, r.Client, ncName, staleVPCs, r.Service.ListVPC())
 	return nil
 }
+
+func (r *NetworkInfoReconciler) getNetworkConfigInfo(networkInfoCR *v1alpha1.NetworkInfo) (commonservice.VPCNetworkConfigInfo, error) {
+	ncName, err := r.Service.GetNetworkconfigNameFromNS(networkInfoCR.Namespace)
+	if err != nil {
+		log.Error(err, "failed to get network config name for VPC when creating NSX VPC", "NetworkInfo", networkInfoCR.Name)
+		return commonservice.VPCNetworkConfigInfo{}, err
+	}
+	nc, _exist := r.Service.GetVPCNetworkConfig(ncName)
+	if !_exist {
+		message := fmt.Sprintf("failed to read network config %s when creating NSX VPC", ncName)
+		log.Info(message)
+		return commonservice.VPCNetworkConfigInfo{}, fmt.Errorf("VPCNetworkConfig %s not found", ncName)
```

Comment:
uisng log.Error? since you removed the log in Line 109

Reply:
updated.